src/metaswitch/clearwater/cluster_manager/main.py

```diff
@@ -170,10 +170,10 @@ def main(args):
         # Don't start any threads as we don't want this box to cluster
         pdlogs.DO_NOT_CLUSTER.log()
     else:
+        # Load the plugins, but don't start them until we've installed the
+        # SIGTERM handler
         for plugin in plugins_to_use:
             syncer = EtcdSynchronizer(plugin, sig_ip, etcd_ip=mgmt_ip)
-            syncer.start_thread()
-
             synchronizers.append(syncer)
             threads.append(syncer.thread)
```

Comment:
Should be fine: https://github.com/Metaswitch/clearwater-etcd/blob/dev/src/metaswitch/clearwater/etcd_shared/common_etcd_synchronizer.py#L250

We setup the thread in `__init__`, and `start_thread` only starts it